### PR TITLE
Restaurant Page with Dynamic Modular JS

### PR DESCRIPTION
**Pull request by anewman15**
In this PR, I created a simple Restaurant Page using ES6 and Webpack.

- Most of the html are generated on the fly with vanilla JS
- There are three tabs in the page, each leading to their respective contents
- Contents of each tab are dynamically generated by their respective JS modules
- SCSS styling and images are processed by Webpack loaders

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @anewman15 👋

**You've done quite well here. 👌**


However, let's make some changes.

- [x] Please check comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -24,9 +24,20 @@
   },
   "homepage": "https://github.com/anewman15/restaurant-page#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^5.0.1",
+    "eslint": "^7.14.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "sass": "^1.29.0",
+    "sass-loader": "^10.1.0",
```
Comment (mikenath223) [inline package.json:35]: - [x] Please include the necessary run commands in scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

File: src/content_contact.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export default function displayContentContact() {
+	resetContentWrapper();
+	const contentWrapper = document.getElementById('content-wrapper');
+	const contentMenu = document.createElement('div');
+	contentMenu.setAttribute('id', 'content-menu');
+	contentMenu.setAttribute('class', 'tab-content');
+
+	contentMenu.innerHTML = `
+                            <div class="address">
+                              <div class="">
```
Comment (mikenath223) [inline src/content_contact.js]: - [x] Kindly use es6 arrow function here. Fix this and other affected functions too.

File: package.json (hunk 1 above)
Comment (anewman15) [reply package.json:35]: Done, thanks for this

File: src/content_contact.js (hunk 2 above)
Comment (anewman15) [reply src/content_contact.js]: Done

**Comment by anewman15** [comment]
Changes made. Thanks, @mikenath223

**Comment by Dandush03** [comment]
# Status: :rocket: Approved
## Keep Up The Good Work!
# Happy Coding :muscle:

**Pull request merged**